src/components/serverPaginatedTable/serverPaginatedTable.js

```diff
@@ -49,6 +51,8 @@ class ServerPaginatedTableView extends React.Component {
     this.setState({ isLoading: true });
     this.fetchData(page * this.state.rowsPerPage, this.state.rowsPerPage, sortOrder).then((res) => {
       this.rowsSelectedTrigger(res);
+      // update columns display true/false depending on onViewColumnsChange
+      this.setUpdatedColumnsDisplay(this.state.updatedColumns);
```

Comment:
Is there a way you can set trigger this up only when view columns is true in the Table props?